spec/features/components_guide_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.feature "Visit Component Guide" do
```

Comment:
Great stuff adding a test for this.

I was a bit surprised to have it as a feature test but since there isn't a concept of requests tests for this project (instead has the older approach of controller tests) then I agree feature is the place to put this stuff.